### PR TITLE
HA improvements

### PR DESCRIPTION
**Pull request by pasinskim**

### CODE REVIEW

**Comment by pasinskim** [comment]
@nickanderson can you take a look at this?

File: cfe_internal/enterprise/ha/ha.cf [hunk 1]
```diff
@@ -49,21 +52,37 @@ bundle agent ha_manage_mp_status_file
...
+     create => "true",
+     perms => mog("0755",$(def.cf_apache_user),$(def.cf_apache_group)),
+     comment => "This directory is used by Mission Portal to store custom
+                 user-defined scripts. Having HA set up we need to synchronize
+                 content of this directory between active and passive hubs. 
+                 In case of no user-scripts being defined directory may not
+                 exist on active hub.
+                 To avoid policy synchronizing content of this directory
+                 being not kept we are creating directory in advance.";
+}
```
Comment (nickanderson) [inline cfe_internal/enterprise/ha/ha.cf]: A+ comment

File: controls/3.6/def.cf [hunk 2]
```diff
@@ -88,6 +88,9 @@ bundle common def
                            # Assume /16 LAN clients to start with
                            "$(sys.policy_hub)/16",
 
+                           # Uncomment below if HA is used
+                           #"@(def.policy_servers)"
```
Comment (nickanderson) [inline controls/3.6/def.cf]: Is it possible to leave this uncommented? At worst, this should end up as an empty list, which should not affect the ACL negatively.

Comment (nickanderson) [inline controls/3.6/def.cf]: It might cause cf_null to be in there so I think it's probably safest to keep it commented as suggested.

**Comment by nickanderson** [comment]
It looks OK to me

**Comment by nickanderson** [comment]
@pasinskim can you send a PR for 3.7.x? My cherry pick failed

**Comment by nickanderson** [comment]
@pasinskim Got it here: https://github.com/cfengine/masterfiles/pull/539

**Comment by nickanderson** [comment]
trigger build

**Pull request merged**